### ISSUE TITLE
[2.4.0+] Excel repair warning

### ISSUE THREAD

**Issue by sylvainbx** [sleede/fab-manager#49]
Opening this as an information notice for users who encounter the following problem:

> When opening an Excel export with Excel 2016 (Windows) or Excel Mac 2011 (OSX), a warning dialog appears, asking to repair the file before opening.

**This does not affect the export content: data integrity is guaranteed** but this can be boring if you export frequently.

This issue is very probably due to a bug in a third party library (axlsx) and was already reported there
(see https://github.com/randym/axlsx/issues/489).

No fixes are expected until axlsx fixes the issue.

As a workaround, you can use [LibreOffice Calc](https://www.libreoffice.org/discover/calc/) ([here in french](https://fr.libreoffice.org/discover/calc/)) to open your exports files without any warning.

**Comment by alexcvcoelho** [comment]
Hi @sylvainbx.

Searching in issues of the axlsx_rails i found this solution [here](https://github.com/straydogstudio/axlsx_rails/issues/55#issuecomment-306756599), that worked for me.

In fact, lib `rails_axlsx` returns a content with encoding error, what I do is use the `write` method that corrects the encode to write to the file.
I am using the following configuration in Gemfile:
```
gem 'rubyzip', '= 1.0.0'
gem 'axlsx', '= 2.0.1'
gem 'axlsx_rails', '=0.4.0'
```
The **old** code:
```Ruby 
File.open(export.file,"w+b") {|f| f.puts content }
 ```
The **new** code:
```Ruby 
File.open(export.file,"w+b") {|f| f.write content }
 ```

**Comment by sylvainbx** [comment]
Hi @alexcvcoelho, I've updated the code to use `f.write` as you suggested. I've also updated the gems dependencies to the last versions. This applies starting from release 4.2.1. Feel free to give it a try and make a feedback here if you have Excel.

**Comment by sylvainbx** [comment]
Office 365 seems to be totally unable to open the exported files.

**Comment by sylvainbx** [comment]
Fixed in [release 5.3.1](https://github.com/sleede/fab-manager/releases/tag/v5.3.1)
